LibTest/io/Link/create_A04_t01.dart

```diff
@@ -38,13 +48,12 @@ main() async {
 }
 
 _main(Directory sandbox) async {
-  if (Platform.isWindows) { //   Directory sandbox = getTempDirectorySync();
-    Link link = new Link(getTempFilePath(parent: sandbox));
-    asyncStart();
-    await link.create(getTempFilePath()).then((Link created) {
-      Expect.fail("Link shouldn't be created on Windows");
-    }, onError: (e) {
-      asyncEnd();
-    });
-  }
+  Directory target = getTempDirectorySync(parent: sandbox);
+  Link link = Link(getTempFilePath(parent: sandbox));
+  asyncStart();
+  await link.create(target.path).then((Link created) {
+    Expect.equals(FileSystemEntityType.directory,
+        FileSystemEntity.typeSync(created.path));
+    asyncEnd();
+  });
```

Comment:
So, just trying to explain the change, this would previously fail because it was executed on Windows in non-privileged mode, and now it's running in a suitable privileged mode (involving "Administrator mode" or a config setting called 'Developer Mode') on Windows?

Reply:
Similarly for some other tests with the same kind of change, and least the next couple of tests.

Reply:
OK, thanks!